src/components/Pill/PillButton/__tests__/PillButton.spec.tsx

```diff
@@ -87,10 +89,10 @@ describe('<PillButton> component', () => {
 
         it('should render correctly', () => {
             expect(view.container).toMatchSnapshot();
-            expect(screen.getByRole('button', { name: '' })).toHaveClass(
+            expect(getButtonByName('')).toHaveClass(
```

Comment:
![image](https://github.com/textkernel/oneui/assets/104063210/c70d97a2-1395-402c-9a4a-0211a0a6b68e)

Reply:
Let's make sure this button is also accessible, and add a name to the button or a label to the svg.

Reply:
Do i understand correct we also add text after svg ? because  <button>Some value</button>  here Some value is name?

Reply:
Do you mean that you want to change the UI by adding text next to the icons? I rather the visual doesn't change. You can add aria-label to the icon, and it will work same like a text when you try to access the "name" of the button

Reply:
When I've added label 
![image](https://github.com/textkernel/oneui/assets/104063210/f5268996-8203-42ab-9668-5e9b46f3c043)
i can't find button

Reply:
Please add the name to the button you are looking for here

Reply:
:+1: